openapitools.json

```diff
@@ -7,7 +7,7 @@
       "cirro": {
         "generatorName": "typescript-fetch",
         "output": "#{cwd}/packages/api-client",
-        "inputSpec": "https://app.cirro.bio/openapi/cirro-data-latest.yml",
+        "inputSpec": "https://dev.cirro.bio/openapi/cirro-data-latest.yml",
```

Comment:
Should this still be app rather than dev?

Reply:
Doesn't matter, we'd most likely be generating off of dev anyways.